README.md

```diff
@@ -44,3 +44,23 @@ It is possible to use a `substra-backend` skaffold file as the source for the ne
 ```
 SUBSTRA_TESTS_SKAFFOLD_FILEPATH=$SUBSTRA_SOURCE/substra-backend/skaffold.yaml make test
 ```
+
+# Test design guidelines
+
+When adding or modifying tests, please follow these guidelines.
+
+The test suite:
+- Must be independent from the substra network
+  - The substra network is started prior to, and independently from the tests being run
+  - The substra network can be running locally / on the cloud
+  - The substra network can be started through docker-compose or through kubernetes
+- It should be possible to run the test suite multiple times without restarting the substra network
+- Must not use hardcoded network configuration settings/values. Use settings files instead (e.g. `values.yaml`)
+- Should target a substra network with at least 2 organisations
+
+Each new test:
```

Comment:
What about "Each test" instead of "Each new test"? I think it would be more accurate, as guidelines also apply to modifying tests.

Reply:
done